modin/_compat/io/__init__.py

```diff
@@ -0,0 +1,12 @@
+# Licensed to Modin Development Team under one or more contributor license agreements.
```

Comment:
This stuff should probably be located under _compat/pandas.

Reply:
Same as in https://github.com/modin-project/modin/pull/4301/#discussion_r918933039 - those functions are for internals.